### PR TITLE
docs(Update README): Add description and edit setup

### PR DESCRIPTION
**Pull request by conor-ward**
Edit description and setup in README

## Proposed Changes

  - Also adds Domain commands to README
  - Changed font size of commands (Can revert this)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,14 +18,20 @@ Node.js 10 or later.
 $ npm i -g swaggerhub-cli
 ```
 ## Setup
+The SwaggerHub CLI can be configured through environment variables or through the [`swaggerhub configure`](#swaggerhub-configure) command. The CLI will look for the following environment variables.
+
+* `SWAGGERHUB_API_KEY` (required) – **Important: please keep this key secure**. This is the SwaggerHub API key the CLI will use for authentication. An API key may be [generated from the user settings page in SwaggerHub](https://app.swaggerhub.com/settings/apiKey).
```
Comment (hkosova) [inline README.md:23]: We don't need to generate the key, it's already there.
```suggestion
* `SWAGGERHUB_API_KEY` (required) – **Important: keep this key secure.** This is the SwaggerHub API key the CLI will use for authentication. You can find your API key on the [user settings page](https://app.swaggerhub.com/settings/apiKey) in SwaggerHub.
```

File: README.md [hunk 2]
```diff
@@ -18,14 +18,20 @@ Node.js 10 or later.
...
+* `SWAGGERHUB_URL` (optional, default is `https://api.swaggerhub.com/`) – Customers with on-premise installations can set to point to their on-premise API, which is `http(s)://{swaggerhub-host}/v1` (do not append a backslash).
```
Comment (hkosova) [inline README.md:24]:
```suggestion
* `SWAGGERHUB_URL` (optional, default is `https://api.swaggerhub.com/`) – Customers with on-premise installations need point this to their on-premise API, which is `http(s)://{swaggerhub-host}/v1` (do not append a backslash). 
```

File: README.md [hunk 3]
```diff
@@ -18,14 +18,20 @@ Node.js 10 or later.
...
+
 ```sh-session
 $ swaggerhub configure
 ? SwaggerHub URL: https://api.swaggerhub.com
 ? API Key: <your-api-key>
 ```
-SwaggerHub URL defaults to the SaaS version, `https://api.swaggerhub.com`. The URL can be reset using the **configure** command
 
-Alternatively, these values can be set using the `SWAGGERHUB_URL` and `SWAGGERHUB_API_KEY` environment variables. These take precedence over config values.
+Environment variables will take precedence over the configuration file created by this command.
```
Comment (hkosova) [inline README.md:34]:
```suggestion
Environment variables take precedence over the configuration file created by this command.
```

File: README.md (hunk 2 above)
Comment (conor-ward) [reply README.md:24]: `need to point this`?

Comment (hkosova) [reply README.md]: @conor-ward yes, thanks

File: README.md (hunk 3 above)
Comment (conor-ward) [reply README.md:34]: Sorry, missed this one

**Pull request merged**